sdk/sdk/src/main/java/org/apache/carbondata/sdk/file/CSVCarbonWriter.java

```diff
@@ -72,6 +90,72 @@ public void write(Object object) throws IOException {
     }
   }
 
+  public static CsvParser buildCsvParser(Configuration conf) {
```

Comment:
can it be private and non-static method?

Reply:
done